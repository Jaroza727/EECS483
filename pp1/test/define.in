#jiaoshou
#JIAOSHOU
#defineJIAOSHOU 2333333
#define JIAOSHOU 2333333
#define   JIAO 23333333
#define JIAO   23333333
#define Jiaoshou 2333333
#define jiaoshou 2333333
string abc = "#JIAO";
int aa = #JIAO;
#define SHOU 			123
#SHOU;
#define JIAO 4567;
#JIAO
#define SHOU
#define SHOU 
#SHOU#SHOU#SHOU1;
#define JIAO */
/*1231313123123#JIAO*/