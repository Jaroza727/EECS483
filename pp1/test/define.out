
*** Error line 1.
*** Invalid # directive



*** Error line 2.
*** Invalid # directive



*** Error line 3.
*** Invalid # directive

 2333333


*** Error line 5.
*** Invalid # directive




*** Error line 7.
*** Invalid # directive



*** Error line 8.
*** Invalid # directive


string abc = "#JIAO";
int aa =   23333333;

            123;

4567;

*** Error line 15.
*** Invalid # directive



1;

